### ISSUE TITLE
roxctl unknown module(s)

### ISSUE THREAD

**Issue by mtcolman** [stackrox/stackrox#9447]
Hello, as per: https://docs.openshift.com/acs/4.2/integration/integrate-with-jira.html#troubleshoot-jira-integration_integrate-with-jira

I'm running:
```
$ roxctl -e https://localhost:8443 central debug log --level Debug --insecure-skip-tls-verify -m notifiers/jira
```

But I receive this error:
```
ERROR:  could not set log level on central: rpc error: code = InvalidArgument desc = Unknown module(s): notifiers/jira: invalid arguments
```

And I can't find what the correct module reference is.

Thanks in advance.

**Comment by janisz** [comment]
What version are you on. I tried to reproduce it and it's working for me. Do you have ACS deployed on localhost or are you using a proxy?

**Comment by mtcolman** [comment]
Hi, v4.2.1. I've got it deployed on OpenShift, accessing via port-forward.

**Comment by mtcolman** [comment]
I've upgraded to v4.3.3 and it works. Is it meant to work in v4.2.1?

```
$ roxctl -e "https://localhost:8443" central debug log --level Debug --insecure-skip-tls-verify -m notifiers/jira
Successfully set log level
```

```
$ roxctl version
4.3.3
```

**Comment by mtcolman** [comment]
However, having now run:
```
$ roxctl -e "https://localhost:8443" central debug log --level Debug --insecure-skip-tls-verify -m notifiers/jira
Successfully set log level

$ roxctl version
4.3.3

$ roxctl -e "https://localhost:8443" central debug dump --insecure-skip-tls-verify
INFO:   Retrieving debug metrics. This may take a couple of minutes...
2024/01/18 15:16:27 [DEBUG] GET https://localhost:8443/debug/dump?logs=false
INFO:   Successfully wrote debug zip file to "stackrox_debug_2024_01_18_15_16_28.zip"

$ roxctl -e "https://localhost:8443" central debug log --level Info --insecure-skip-tls-verify
Successfully set log level
```

I've checked the contents of the zip:
![image](https://github.com/stackrox/stackrox/assets/33348891/26b7505e-fc86-47e2-84ff-084348abcf07)

And looked in `notifiers.json` (in absence of any other guidance) and it just says `null`.

**Comment by janisz** [comment]
@mtcolman Thank you for your report. I was able to reproduce it on 4.2.1. I'll investigate why this happens.

The JSONs you are seeing is the configuration not related to logs that you set with log level, I need to verify if they could be `null` or should be empty. Did you have any notifier configured?

**Comment by mtcolman** [comment]
Thank you. No I don't have other notifiers configured.

**Comment by mtcolman** [comment]
@janisz will the docs be updated to reflect this? Shall I close this ticket?

**Comment by janisz** [comment]
I created a PR for docs. Thanks for reporting!

PR that introduced this change was released in 4.3 
- https://github.com/stackrox/stackrox/pull/8199
- https://github.com/openshift/openshift-docs/pull/70684
